codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/test-abstract-async-client-class.java

```diff
@@ -643,15 +657,12 @@ public <ReturnT> CompletableFuture<ReturnT> streamingOutputOperation(
     }
 
     @Override
-    public void close() {
-        delegate.close();
+    public final String serviceName() {
+        return SERVICE_NAME;
```

Comment:
should we delegate the serviceName method?

Reply:
updated to return delegate.serviceName()